packages/material-ui/src/Menu/Menu.test.js

```diff
@@ -113,11 +116,11 @@ describe('<Menu />', () => {
     assert.strictEqual(wrapper.find(Popover).props().getContentAnchorEl != null, true);
   });
 
-  it('should pass onClose prop to Popover', () => {
-    const fn = () => {};
-    const wrapper = mount(<Menu {...defaultProps} onClose={fn} />);
-    assert.strictEqual(wrapper.find(Popover).props().onClose, fn);
-  });
+  // it('should pass onClose prop to Popover', () => {
+  //   const fn = () => {};
+  //   const wrapper = mount(<Menu {...defaultProps} onClose={fn} />);
+  //   assert.strictEqual(wrapper.find(Popover).props().onClose, fn);
+  // });
```

Comment:
Should this be uncommented and fixed?

Reply:
Verlon removed this test. It was a while ago, we should see if it will work now that we have made changes.

Reply:
I'll take a look and see.

Reply:
It works.